store/sql_store.go

```diff
@@ -215,15 +140,15 @@ func setupConnection(con_type string, driver string, dataSource string, maxIdle
 	if err != nil {
 		l4g.Critical(utils.T("store.sql.open_conn.critical"), err)
 		time.Sleep(time.Second)
-		panic(fmt.Sprintf(utils.T("store.sql.open_conn.critical"), err.Error()))
+		os.Exit(101)
```

Comment:
If we don't name these we are going to loose count. Someone is going to do 123 then 123 again somewhere else.